### PR TITLE
feat: Add Paramater to update graphiql example query

### PR DESCRIPTION
**Pull request by LucasHartridge**
## Summary

Hello Team :wave:  I have recently started using Strawberry in order to build a GraphQL API using FastAPI. Despite GraphiQL is disabled on production environments, I find it useful to deploy it into a Dev environment so Project Managers and other members can verify the data of the API and play with it

I could not find any way of updating the default message of the GraphiQL, so I decide to put a PR on the repo. Please let me know if I am missing something or if there is any way that I can improve on it. For the use case that I described, I find it very useful since I can adapt the message to the final consumers of the interface

I know a simple way is to inherit the GraphQLRouter and overwrite the render_graphiql but I went through the code and did not seem to be a hard change to achieve 

## Description

- In order to achieve this change I added an argument on the **get_graphiql_html** function that is implemented by all the integrations that Strawberry supports that will receive the new message
- On the same function I had to update the template in order to also update the constant that existed on the static/graphiql.html so in case nothing is passed, it will use the default message that was already defined as a const on the HTML
- The last step was to go integration by integration and update the constructor to receive this new argument

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [X] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

*

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the CONTRIBUTING document.
- [X] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

## Comments
I had run all the unit tests and all of them are passing, but I believe I need to improve the coverage on Django and the others integrations

I will come back later to see if I can achieve it, I need more time but wanted to open the PR in the mid-time to know your thoughts 

Also, I believe in case this wants to be merged, the documentation should be updated in order to indicate the new argument and how it works

Thanks

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

Being able to update the GraphiQL Default Example Query.



Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Lucas Hartridge for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

File: strawberry/aiohttp/views.py [hunk 1]
```diff
@@ -98,6 +98,7 @@ def __init__(
             GRAPHQL_WS_PROTOCOL,
         ),
         connection_init_wait_timeout: timedelta = timedelta(minutes=1),
+        graphiql_example_query: Optional[str] = None,
```
Comment (jcunhafonte) [inline strawberry/aiohttp/views.py:101]: Should this param be only named `example_query`?

File: strawberry/chalice/views.py [hunk 2]
```diff
@@ -76,7 +78,7 @@ def render_graphiql(self, request: Request) -> Response:
         Returns:
             The GraphiQL html page as a string
         """
-        return get_graphiql_html(subscription_enabled=False)  # type: ignore
+        return get_graphiql_html(subscription_enabled=False, example_query=self.graphiql_example_query)  # type: ignore
```
Comment (jcunhafonte) [inline strawberry/chalice/views.py:81]: Same as above:
Should this param be only named `example_query`?

**Comment by jcunhafonte** [comment]
I am absolutely thrilled to express my heartfelt appreciation for this PR. This contribution is nothing short of exceptional and has been long-awaited.

Thank you for making this long-desired feature a reality, and for your invaluable contribution to the project. Kudos to you @LucasHartridge! 👏🙌

File: strawberry/chalice/views.py (hunk 2 above)
Comment (LucasHartridge) [reply strawberry/chalice/views.py:81]: Yes, I will update it

**Comment by DoctorJohn** [comment]
Hey everyone, I'm currently reviewing #2842 which make GraphiQL store the query in the URL search params. This allows anyone to share their queries easily. In some way #2842 could replace this PR: Instead of providing a custom example query in code one could share a custom example query using a URL. It's just a thought tho. I'll happily discuss and review this PR too, just let me know what you think :)

**Comment by LucasHartridge** [comment]
@DoctorJohn I totally agree with you, this was the goal of the PR, but if we can easily achieve it by a query param, we can save all the risk of updating all the files just to add it to the library 

Thanks I will close the PR and we will wait for https://github.com/strawberry-graphql/strawberry/pull/2842 to be merged

**Pull request closed without merging**